### PR TITLE
Add skeleton for personal portfolio page (elton) 

### PR DESCRIPTION
**Pull request by notle1706**
Week 7 tp

### CODE REVIEW

**Review by Jo3LW — APPROVED** [review]
LGTM :)

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,15 +9,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Elton Chua
```
Comment (quannam0124) [inline docs/AboutUs.md:12]: LGTM Elton! Nice photo too!

**Review by quannam0124 — COMMENTED** [review]
LGTM!

**Review by quannam0124 — APPROVED** [review]
LGTM & Nice photo!

**Pull request merged**